### PR TITLE
add autoshutdown tags to vm

### PR DESCRIPTION
**Pull request by NickAzureDevops**
### Jira link (https://tools.hmcts.net/jira/browse/DTSPO-18289)

add auto shutdown tag to bastion sbox and nonprod 

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Comment by NickAzureDevops** [comment]
Pipeline run: https://dev.azure.com/hmcts/PlatformOperations/_build/results?buildId=629208&view=logs&j=87de6e83-4483-545f-5ee4-dd646a473fc0&t=9f07a130-add3-5319-3995-dd3dcc1a42ca&l=192

File: terraform/environment/sbox/21-bastion.tf [hunk 1]
```diff
@@ -38,4 +38,5 @@ module "bastion-secops-sbox" {
   # Tenable Nessus
   nessus_server     = "nessus-scanners-sbox000006.platform.hmcts.net"
   nessus_key_secret = "nessus-agent-key-sbox"
+  common_tags       = merge(var.common_tags, local.auto_shutdown_common_tags)
```
Comment (bancey) [inline terraform/environment/sbox/21-bastion.tf:41]: var.common_tags here is an empty map, this should reference a call to the common tags module

File: terraform/environment/nonprod/21-bastion.tf [hunk 2]
```diff
@@ -18,4 +18,5 @@ module "bastion-nonprod" {
   # Tenable Nessus
   nessus_server     = "nessus-scanners-nonprod000005.platform.hmcts.net"
   nessus_key_secret = "nessus-agent-key-nonprod"
+  common_tags       = merge(var.common_tags, local.auto_shutdown_common_tags)
```
Comment (bancey) [inline terraform/environment/nonprod/21-bastion.tf:21]: var.common_tags here is an empty map, this should reference a call to the common tags module

**Review by bancey — CHANGES_REQUESTED** [review]

**Comment by NickAzureDevops** [comment]
it is all fixed now. found out the tags were in the tag module but just set as false for auto shutdown

**Comment by NickAzureDevops** [comment]
Run https://dev.azure.com/hmcts/PlatformOperations/_build/results?buildId=631308&view=logs&j=87de6e83-4483-545f-5ee4-dd646a473fc0&t=9f07a130-add3-5319-3995-dd3dcc1a42ca

**Review by bancey — APPROVED** [review]

**Pull request merged**